integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -13,11 +13,8 @@ setup() {
 	clean_env
 }
 
-@test "[volume mkdir] create a directory in an aerospike container" {
-	tmpdir=$(mktemp -d)
-	docker run --rm --runtime=$RUNTIME -d -v $tmpdir:/opt/aerospike/data aerospike/aerospike-server timeout 5 /entrypoint.sh asd
-	[ -n ${tmpdir} ] || false
-	rm -rf $tmpdir
+@test "[display configuration] display port in configuration file in an aerospike container" {
+	docker run --rm --runtime=$RUNTIME aerospike cat /etc/aerospike/aerospike.conf | grep port
```

Comment:
I think that would be better to have a test where data is inserted in a aerospike server

```
docker run --rm --runtime=$RUNTIME  -d --name aerospike aerospike/aerospike-server
sleep 10
docker run --rm --runtime=$RUNTIME -it aerospike/aerospike-tools aql -h  $(docker inspect -f '{{.NetworkSettings.IPAddress }}' aerospike) -c "insert into test.foo (PK, foo) values ('123','any') ; select * from test.foo"
docker rm -f aerospike
```

@GabyCT what do you think?